challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |string| string.capitalize}
```

Comment:
There's a shorthand that's equivalent but a bit "magical": `input.map(&:capitalize)`

The shorthand is typically preferred.

Check out http://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/ or Google "ruby symbol to_proc" for others.